src/app/pages/MediaAssetPage/index.jsx

```diff
@@ -30,14 +33,19 @@ import {
   getAboutTags,
 } from '#lib/utilities/parseAssetData';
 
+import { RequestContext } from '#contexts/RequestContext';
+
 // Page Handlers
 import withContexts from '#containers/PageHandlers/withContexts';
 import withPageWrapper from '#containers/PageHandlers/withPageWrapper';
 import withError from '#containers/PageHandlers/withError';
 import withLoading from '#containers/PageHandlers/withLoading';
 import withData from '#containers/PageHandlers/withData';
 
+const isLegacyAsset = url => url.split('/').length > 7;
```

Comment:
is this really the best way to tell if is a legacy asset? if it is then carry on!

Reply:
It was a choice between the number of slashes and checking whether it contained a date:

uzbek.multimedia.2013.12.131221_sectarian_war_bowen.page

pidgin.media_asset.51536047.page

Counting the slashes was simpler but I would be eager to hear other suggestions

Reply:
Feels wrong but it does seem to be the best way - Jane checked with Dan and inferring it from the URL is what he suggested as well